code-orange/ViewController.swift

```diff
@@ -9,17 +9,50 @@
 import UIKit
 import GoogleMaps
 
-protocol CurrentLocationProvider: class {
+protocol LocationProvider: class {
```

Comment:
@renenel can we move the protocol to a different file in the model layer?

Reply:
done!